package.json

```diff
@@ -17,7 +17,8 @@
     "release:create": "create-release",
     "release": "./scripts/publish-release.sh",
     "start:playground": "npm run build && node test/app.js",
-    "start": "npm run docs && npx gulp default",
+    "start:browser-sync": "browser-sync start --proxy 'localhost:4000' --files 'docs/_site/**/*' --reloadDelay '2000'",
+    "start": "npm run docs:compile && cd docs && bundle exec jekyll serve --incremental --config _config.yml,_config-library.yml",
```

Comment:
If the `start` script already does hot reloading, what is the `start:browser-sync` doing?  Maybe I just don't understand what browser-sync does. 😄

Reply:
Browser sync is pretty cool! It lets you open localhost in several windows, even across browsers, and keeps your url in sync. So let's say I'm looking at the homepage on chrome, safari and firefox in different windows. I click on actionbar on chrome, safari and firefox will update their url as well. I can easily compare what everything looks like across different browsers without any overhead. I also played around using it to check responsiveness. 

It was built in under the hood before, I think I (and probably you) just didn't know to take advantage of it before :)

Reply:
Nice -- I didn't know that. 👍